src/screens/select.tsx

```diff
@@ -1,12 +1,11 @@
-import React, { useState } from 'react';
```

Comment:
I've already left comments about this file on the other PR. You probably won't have time to do this, but as a next step I'd want to pull stuff out of it since it's getting quite complicated - your code to fetch from the API can be defined in a helper file, and a lot of these inputs could be pulled out into subcomponents

Reply:
Yes